### PR TITLE
[FLINK-13330][table-planner-blink] Remove unnecessary to reduce testing time in blink

### PR DESCRIPTION
**Pull request by JingsongLi**
## What is the purpose of the change

Some tests are unnecessary.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/runtime/utils/StreamingWithAggTestBase.scala [hunk 1]
```diff
@@ -63,9 +63,11 @@ object StreamingWithAggTestBase {
 
   @Parameterized.Parameters(name = "LocalGlobal={0}, {1}, StateBackend={2}")
   def parameters(): util.Collection[Array[java.lang.Object]] = {
+    // To avoid too long testing time, we not test all HEAP_BACKEND.
+    // If HEAP_BACKEND has changed, please open it manually for testing.
```
Comment (twalthr) [inline flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/runtime/utils/StreamingWithAggTestBase.scala]: Comments like this get lost in the code base. Tests like this should be moved to a special manual tests package.

File: flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/sort/BinaryExternalSorterTest.java [hunk 2]
```diff
@@ -76,11 +75,14 @@ public BinaryExternalSorterTest(
...
-				new Boolean[]{false, true},
-				new Boolean[]{true, false},
-				new Boolean[]{true, true});
+		// To avoid too long testing time, we only tested the default configuration.
+		// If BinaryExternalSorter has changed, please open it manually for testing.
+//		return Arrays.asList(
+//				new Boolean[]{false, false},
+//				new Boolean[]{false, true},
+//				new Boolean[]{true, false},
+//				new Boolean[]{true, true});
```
Comment (wuchong) [inline flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/sort/BinaryExternalSorterTest.java]: Manually testing is hard to guarantee the correctness. I think we need to find a way to add this to nightly tests in this pull request.

**Comment by JingsongLi** [comment]
Thanks @twalthr @wuchong , this tests should move to crontab and daily run. Every tests is important and precious. If we remove them, there maybe some bugs in future.

**Comment by JingsongLi** [comment]
I close this now, @wuchong will do some work to extract them to daily run.

**Pull request closed without merging**